server/models/handlers/appointment.js

```diff
@@ -48,8 +48,39 @@ async function cancelWorkerAppointments(workerId, specificDate) {
     }
 }
 
+async function cancelSpecificAppointment(appointmentId) {
+    // If this method is somehow called without specifying values for the required parameters, false is returned.
+    if (_.isNil(appointmentId)) return false;
+    if (!_.isNumber(appointmentId)) return false; 
+
+    try {
+        //Check if appt exists
+        const doesAppointmentExist = await appointmentModel.appointmentExists(appointmentId);
+        if (doesAppointmentExist == false) return false;
+
+        //low priority, check if appt is not cancelled 
```

Comment:
Hi Peter,
Refer to the above suggestion for revising the method to return the actual appointment. Given that the actual appointment object is returned, this check can easily leverage the status value in the appointment object and the relevant AppointmentStatus constants value.

Reply:
See comment above